admin_manual/installation/examples/lets_encrypt/nginx.rst

```diff
@@ -0,0 +1,183 @@
+Setup NGINX
+===========
+
+The following is an example setup process for NGINX, please adapt it to your exact needs.
+
+NGINX ssl_dhparam
+-----------------
+
+If not already present, add an `ssl_dhparam`_ directive and a new certificate with stronger keys (which improves `forward secrecy`_). 
+The OpenSSL command may take a while to complete, so please be patient. 
+You can place the certificate into any directory you choose.
```

Comment:
- Use the same text here as in apache ``Apache SSLOpenSSLConfCmd DHParameters``:
"You can place the certificate into any directory of your choice.
We recommend ``/etc/nginx/`` in this guide, just for sakes of simplicity."